src/TTImp/TTImp/Traversals.idr

```diff
@@ -94,8 +94,8 @@ parameters (f : RawImp' nm -> RawImp' nm)
     = f $ ILam fc rig (mapPiInfo pinfo) x (mapTTImp argTy) (mapTTImp lamTy)
   mapTTImp (ILet fc lhsFC rig n nTy nVal scope)
     = f $ ILet fc lhsFC rig n (mapTTImp nTy) (mapTTImp nVal) (mapTTImp scope)
-  mapTTImp (ICase fc t ty cls)
-    = f $ ICase fc (mapTTImp t) (mapTTImp ty) (assert_total $ map mapImpClause cls)
+  mapTTImp (ICase fc opts t ty cls)
+    = f $ ICase fc opts (mapTTImp t) (mapTTImp ty) (assert_total $ map mapImpClause cls)
```

Comment:
Shouldn't `libs/base/Language/Reflection/TTImp.idr` be changed accordingly?

Reply:
Ah, yes. I always forget about that. Thanks!